lib/commands/migrate.js

```diff
@@ -1,42 +1,81 @@
 "use strict";
 
 const fs = require("fs");
+const path = require("path");
 const chalk = require("chalk");
 const diff = require("diff");
 const inquirer = require("inquirer");
 const PLazy = require("p-lazy");
 const Listr = require("listr");
 
-const validate = require("webpack").validate;
-const WebpackOptionsValidationError = require("webpack")
-	.WebpackOptionsValidationError;
+const { validate } = require("webpack");
+const { WebpackOptionsValidationError } = require("webpack");
 
 const runPrettier = require("../utils/run-prettier");
 
 /**
-*
-* Runs migration on a given configuration using AST's and promises
-* to sequentially transform a configuration file.
-*
-* @param {String} currentConfigPath - Location of the configuration to be migrated
-* @param {String} outputConfigPath - Location to where the configuration should be written
-* @param {Object} options - Any additional options regarding code style of the written configuration
+ *
+ * Runs migration on a given configuration using AST's and promises
+ * to sequentially transform a configuration file.
+ *
+ * @param {Array} args - Migrate arguments such as input and
+ * output path
+ * @returns {Function} Runs the migration using the 'runMigrate'
+ * function.
+ */
 
-* @returns {Promise} Runs the migration using a promise that will throw any errors during each transform
-* or output if the user decides to abort the migration
-*/
+module.exports = function migrate(...args) {
+	const filePaths = args ? args.slice(3) : null;
+	if (!filePaths.length) {
```

Comment:
Double check here
```js
if (!filePaths || !filePaths.length)
```

Reply:
Gotcha

Reply:
I actually tested this manually and we don't need that double check. It's going to be an empty array anyway if filepaths is empty

Reply:
So remove the ternary operator or fall  back the assignment to an empty array. It might work but people that see the code might think that is wrong. Or add a comment

Reply:
Done